spec/requests/api/v2/providers/courses_spec.rb

```diff
@@ -18,20 +19,33 @@
              start_date: Time.now.utc,
              site_statuses: [create(:site_status, :findable, :with_any_vacancy, :applications_being_accepted_now)])
     }
-    let(:provider) { create(:provider, course_count: 0, courses: [findable_open_course]) }
+    let(:provider) {
+      create(:provider,
+             course_count: 0,
+             courses: [findable_open_course],
+             organisations: [organisation])
+    }
     subject { response }
 
     before do
       get "/api/v2/providers/#{provider.provider_code}/courses",
           headers: { 'HTTP_AUTHORIZATION' => credentials }
     end
 
-    context 'when unauthorized' do
+    context 'when unauthenticated' do
```

Comment:
`unauthenticated` vs `unauthorized` you sure as http code returns `unauthorized`

Reply:
I **think** this is right – there isn't a separate `unauthorized` HTTP status code to my knowledge.

As I understand it:

* an unauthenticated user isn't someone who we have on the system
* an unautorised user is someone who we have on the system, but shouldn't have access to the thing they're trying to access

Reply:
Ya, I think you're right. And as you point out, @benilovj, "Authorisation" seems to be the catch-all term, but if we can disambiguate, that seems to be the correct definition.